src/ui/index.ts

```diff
@@ -13,10 +13,10 @@ function uiReducer(
   return immerProduce(state, (draftState: UIState) => {
     switch (action.type) {
       case actionsList.userMenuActive:
-        draftState.userMenuActive = action.payload;
+        draftState.userMenuActive = action.payload as boolean;
```

Comment:
I've created a follow-up issue for myself https://github.com/canonical-web-and-design/juju-squad/issues/1648